components/03-modules/dashboard/actions-toolbar/actions-toolbar.config.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  label: 'Actions toolbar',
+  name: 'dashboard-actions-toolbar',
+  context: {
+    class: '',
```

Comment:
I think we don't need empty attributes in config

Reply:
removed